docs/browser.md

```diff
@@ -20,16 +20,34 @@ approach comes with a lot of advantages, but also has some drawbacks:
 
 - Longer warm-up time
 - Increased risk of trade collisions
-- Consumes more end-user resources 
+- Consumes more end-user resources
 
-## Installing
+## @0x/mesh-browser
 
 For your convenience, we've published an NPM package called `@0x/mesh-browser`
-which includes the WebAssembly bytecode and a lightweight wrapper around it. You
-install this package in exactly the same way as any other NPM package and using
-it feels exactly like using a native TypeScript/JavaScript library.
+which includes the WebAssembly bytecode and a lightweight wrapper around it. To
+find more information on optimizing the startup time of using Mesh in the browser,
+refer to the next section on the `@0x/mesh-browser-lite` package. This package is
+used exactly the same way as any other NPM package and using it feels exactly
+like using a native TypeScript/JavaScript library.
 
-To install the NPM package, simply run:
+## @0x/mesh-browser-lite
+
+We've published a lightweight version of the `@0x/mesh-browser` package -- the
+`@0x/mesh-browser-lite` package that provides an identical abstraction around a
+browser-based Mesh node without requiring that Wasm bytecode be bundled with the
+rest of the webpage's code. Additionally, this package makes use of the
+WebAssembly's streaming functionality, which could provide a speedup to load
+times for users of browser-based Mesh.
+
+Using this package is a bit more complicated than using the `@0x/mesh-browser` package.
+This package provides an API that will create a Mesh wrapper using a provided URL
+or Response object that serves the WebAssembly bytecode that will be used to run a
+Mesh node.
```

Comment:
Can we be more specific here? Something like:

It requires you to obtain a copy of the WebAssembly bytecode from [the releases page](https://github.com/0xProject/0x-mesh/releases) and serve it on your own server or a CDN.  It also requires you to make one additional function call to either `loadMeshStreamingWithURLAsync` or `loadMeshStreamingAsync` in your code before initializing Mesh.